From 36d2686de3bdf9d58f8bc55c239513655fd5dbd2 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 25 Oct 2023 17:40:00 +0200
Subject: [PATCH 385/841] fix(st-clock): make ops_array_mp13 static

This corrects sparse warning:
drivers/st/clk/clk-stm32mp13.c:1846:28: warning: symbol
 'ops_array_mp13' was not declared. Should it be static?

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 87c580c8444505f7f45c88c57903682afb79135f fix(st-clock): make ops_array_mp13 static]
Change-Id: I667fe67c319f0f5e9b7698128c91abe8c8f68707
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/338362
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/clk/clk-stm32mp13.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/st/clk/clk-stm32mp13.c b/drivers/st/clk/clk-stm32mp13.c
index f0694010c..d0dca0762 100644
--- a/drivers/st/clk/clk-stm32mp13.c
+++ b/drivers/st/clk/clk-stm32mp13.c
@@ -1843,7 +1843,7 @@ enum {
 	MP13_LAST_OPS
 };
 
-const struct stm32_clk_ops *ops_array_mp13[MP13_LAST_OPS] = {
+static const struct stm32_clk_ops *ops_array_mp13[MP13_LAST_OPS] = {
 	[NO_OPS] =  NULL,
 	[FIXED_FACTOR_OPS] = &clk_fixed_factor_ops,
 	[GATE_OPS] = &clk_gate_ops,
-- 
2.47.2

